docs/guide/index.asciidoc

```diff
@@ -140,30 +140,15 @@ you can use the {apm-server-ref}/overview.html#security[secret token and TLS] to
 [float]
 ==== Install the Kibana dashboards
 APM Server comes with predefined Kibana dashboards for the APM data.
-To install the Kibana dashboards for the 6.x releases,
-run the following command:
+To install the Kibana dashboards run the following command:
 
 [source,bash]
 ----------------------------------
-curl -X POST http://localhost:5601/api/kibana/dashboards/import \
-  -H 'Content-type: application/json' \
-  -H 'kbn-xsrf: true' \
-  -d @./kibana/default/dashboard/apm-dashboards.json
-----------------------------------
-
-If you are running Kibana 5.6, you need to download the dashboard file first and then load it:
-
-[source,bash]
-----------------------------------
-curl -o apm-dashboard.json https://raw.githubusercontent.com/elastic/apm-server/master/_meta/kibana/5.x/dashboard/apm-dashboards.json
-curl -X POST http://localhost:5601/api/kibana/dashboards/import \
-  -H 'Content-type: application/json' \
-  -H 'kbn-xsrf: true' \
-  -d @./apm-dashboards.json
+./apm-server setup
```

Comment:
Running this command sets up template and dashboards. I would say that is the command people should run, but perhaps we should add above that it installes template and dashboards?